lampod/src/liquidity.rs

```diff
@@ -0,0 +1,35 @@
+use std::sync::Arc;
+
+use lampo_common::keys::LampoKeysManager;
+use lightning_liquidity::{events::Event, LiquidityManager};
+
+use crate::{chain::LampoChainManager, ln::LampoChannel};
+
+type LampoLiquidity =
+    LiquidityManager<Arc<LampoKeysManager>, Arc<LampoChannel>, Arc<LampoChainManager>>;
+
+struct LampoLiquiditySource {
+    lampo_liquidity_manager: Arc<LampoLiquidity>,
+}
+
+impl LampoLiquiditySource {
+    fn new(liquidity: Arc<LampoLiquidity>) -> Self {
+        LampoLiquiditySource {
+            lampo_liquidity_manager: liquidity,
+        }
+    }
+
+    pub fn clean_events(&self) {
+        self.lampo_liquidity_manager.get_and_clear_pending_events();
+    }
```

Comment:
are you sure that this function does not need the return type?

Reply:
Thanks, changing